libs/features/mas/web-components/src/merch-card-collection.js

```diff
@@ -283,7 +283,7 @@ export class MerchCardCollection extends LitElement {
             <sp-action-menu
                 id="sortButton"
                 size="m"
-                @change="${this.sortChanged}"
+                onchange="${this.sortChanged}"
```

Comment:
why this change?  this seems wrong to me.

Reply:
This change got there by mistake, thanks for noticing!